flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -157,10 +157,13 @@ class CDUAocOfpData {
             return mcdu.getDelayBasic();
         };
         mcdu.onLeftInput[4] = async () => {
-            if (currentBlockFuel) {
-                loadFuel(mcdu, updateView);
+            const isOnGround = SimVar.GetSimVarValue('L:A32NX_RCDR_GROUND_CONTROL_ON', 'bool');
```

Comment:
how about using the ```SIM ON GROUND``` simvar instead. That's the one used for the EFB as well.

Reply:
rework done.

Reply:
@sidnov @erso44 What about using `Simplane.getIsGrounded()`?

Reply:
I am currently also adding GPS GROUND SPEED it should be enough.

Reply:
> 
> 
> I am currently also adding GPS GROUND SPEED it should be enough.

My point is code consistency. The method I suggested is usually used to test if the aircraft is on ground.

Reply:
Also before you decide to merge my PR I have to change the changelog which is wrong. I don't want to commit to much.